src/main/AndroidManifest.xml

```diff
@@ -39,6 +39,15 @@
     <uses-sdk android:minSdkVersion="8"/>
 
     <application>
+        <!--suppress AndroidDomInspection -->
+        <activity
+            android:name="org.acra.CrashReportDialogImpl"
```

Comment:
Example: 
This is an (admittedly useless) simple implementation using a different superclass
```java
@AcraDialog
public abstract class MyCrashReportDialog extends AppCompatActivity implements ICrashReportDialog {
    @Override
    public void init(@Nullable Bundle savedInstanceState) {
        Toast.makeText(this, "This is a CrashReportDialog", Toast.LENGTH_SHORT).show();
        sendCrash(null, null);
    }
}
```

The compiler will now generate this for it: 
```java
public final class CrashReportDialogImpl extends MyCrashReportDialog {
    private final CrashReportDelegate delegate;

    public CrashReportDialogImpl() {
        delegate = new CrashReportDelegate(this);
    }

    @Override
    public void onCreate(Bundle arg0) {
        super.onCreate(arg0);
        if(delegate.loadFromIntent(getIntent())) {
            init(arg0);
        }
        else {
            finish();
        }
    }

    @Override
    public void cancelReports() {
        delegate.cancelReports();
    }

    @Override
    public void sendCrash(@Nullable String arg0, @Nullable String arg1) {
        delegate.sendCrash(arg0, arg1);
    }

    @Override
    public ACRAConfiguration getConfig() {
        return delegate.getConfig();
    }

    @Override
    public Throwable getException() {
        return delegate.getException();
    }
}
```